components/user/network.js

```diff
@@ -37,4 +37,18 @@ router.post('/',
       })
   })
 
+router.delete('/:id', isAdmin, async (req, res) => {
+  const { params: { id } } = req
+  try {
+    const deleted = await controller.deleteUser(id)
+    if (!deleted) {
+      res.status(404).send({ Error: `A user with that ${id} was not found` })
+      return
+    }
+    res.status(204).send()
```

Comment:
copiar formato de response

Reply:
@dvelezroman  done ... for example
![image](https://user-images.githubusercontent.com/10531037/140103794-e298411c-560c-45fa-b678-dfb9a3b3f6f4.png)